### DISCUSSION TITLE
GitHub Container Registry

### DISCUSSION THREAD

**Discussion by LeviPesin** [docker/extension-ideas#137] [Ideas]
Please add an extension for pulling/pushing images from/to GitHub Container Registry.

**Comment by kornbred** [comment]


**Reply by amyb12345** [reply]
@kornbred Hi! I’m the product manager for Docker Extensions. Extensions in Docker Desktop are free to use for all subscription levels ( free through business). Some extensions may require you to sign into the other companies service, but we do not charge anything to use extensions.  I’m only aware of one GitHub related extension that’s under development by a community member, but it didn’t require any payment to use. 

Are you able to share where to find this extension so we can investigate further? Extensions for Docker Desktop do not require any payment to Docker to install or use. They are free to developers using our products.

**Reply by amyb12345** [reply]
@LeviPesin there is an open source extension under development by a community member for this functionality! Give this extension a try and feel free to contribute to it to add more functionality!  https://github.com/peacecwz/github-registry-docker-desktop-extension

**Reply by kornbred** [reply]
@amyb12345, then I apoligize and must have read something completely wrong. I'm sorry.

**Reply by amyb12345** [reply]
@kornbred no problem! We really are interested in helping developers improve their workflows with extensions.  If you do find anything like this saying that they are paid, please let us know where so we can get it corrected!  We’d love to see more developers using them and definitely don’t want to see anyone not try them because there is false info out there about a fee to use.

**Reply by LeviPesin** [reply]
@amyb12345 Thank you, that seems to work for me!
